cola/linalg/eig/power_iteration.py

```diff
@@ -72,5 +72,9 @@ def cond(state):
         return (i < max_iter) & (err(state) > tol)
 
     while_loop, infodict = xnp.while_loop_winfo(err, tol, pbar=pbar)
-    _, v, _, emax, _ = while_loop(cond, body, (0, v, v, 10., 1.))
+    i0 = xnp.array(0, dtype=xnp.int64, device=A.device)
```

Comment:
why change this?

Reply:
Because the dtype behavior can be erratic (this is a fix to a test). The problem with creating arrays like 10. is that they infer the global dtype which would be float64 if for some reason that is activated. This might defer from the actual dtype that we want to use.

Reply:
ok makes sense